src/model/entity/room_test.go

```diff
@@ -0,0 +1,96 @@
+package entity
+
+import "testing"
+
+func TestNewRoom(t *testing.T) {
+	tests := []struct {
+		name     string
+		roomType RoomType
+		shape    Box
+		portal   *Box
+		want     Room
+	}{
+		{
+			name:     "New room",
+			roomType: RoomTypeFinish,
```

Comment:
Вот тут и дальше нужно обозначить какая именно комната тестируется, чтобы отличать один тест-кейс от другого.


```suggestion
			name:     "New finish room",
			roomType: RoomTypeFinish,
```

Reply:
Поправила